router/router.go

```diff
@@ -676,8 +676,14 @@ func (worker *workerT) processDestinationJobs() {
 								if worker.rt.transformerProxy {
 									jobId := destinationJob.JobMetadataArray[0].JobID
 									pkgLogger.Debugf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Request started`, worker.rt.destName, jobId)
+									proxyReqparams := &transformer.ProxyRequestParams{
+										DestName:     worker.rt.destName,
+										JobId:        jobId,
+										ResponseData: val,
+									}
 									rtlTime := time.Now()
-									respStatusCode, respBodyTemp = worker.rt.transformer.ProxyRequest(ctx, val, worker.rt.destName, jobId)
+									// Should we change this context to sendCtx and probably adjust the timeout for the context ?
```

Comment:
Yes I think it makes much more sense to use `sendCtx` than the global `ctx` over here

Reply:
It was changed as part of some kind of review comment or something a long-time ago. Did we see any performance impact due to using `DetectContentType` in general ?

Reply:
Done